activiti-cloud-services-org/activiti-cloud-services-org-rest/src/main/java/org/activiti/cloud/services/organization/rest/api/ProjectRestApi.java

```diff
@@ -86,8 +85,7 @@
             value = "List projects",
             notes = "Get the list of available projects. " +
                     "Minimal information for each project is returned.",
-            produces = APPLICATION_JSON_VALUE,
-            response = AlfrescoProjectsPage.class)
```

Comment:
@constantin-ciobotaru I don't think this is necessary anymore, so I'm removing it. Let me know if it causes any issue.